### DISCUSSION TITLE
Minor improvement of the GUI

### DISCUSSION THREAD

**Discussion by UliGit** [keepassxreboot/keepassxc#9195] [Ideas]
Hi, 
first of all KeepassXC is really a great tool. I switched from Keepass last year.
The only thing which I found a little better in Kepass was - and this is not only because I'm not used to the approach in KeepassXC, because even after some month I have the same opinion - is that the click on the name or password attribute directly in the list view of the entries to copy its content into the clipboard works smoother and faster than moving to the menue and searching the right button. It is mch more intuitive.
But this is nice to have only.

**Comment by droidmonkey** [comment]
Enable that feature in the application settings, security tab

**Comment by UliGit** [comment]
Oh, sorry for that. 
I already have the best tool with all my favorite features and I haven't noticed it for months.
